smoketest.sh

```diff
@@ -31,7 +31,7 @@ echo " Done."
 
 echo -n "Backing up img_state1.img. "
 ln -sf img_state1.img ../original
-$BACKY backup test
```

Comment:
Please add a test that shows that backy will refuse to create an unknown and non-manual tag.